godot-core/src/obj/script.rs

```diff
@@ -459,18 +478,22 @@ impl<'a, T: ScriptInstance> DerefMut for SiMut<'a, T> {
 
 // Encapsulate BoundedPtrList to help ensure safety.
 mod bounded_ptr_list {
+    #[cfg(before_api = "4.3")]
     use std::collections::HashMap;
+    #[cfg(before_api = "4.3")]
```

Comment:
I wonder if the whole `bounded_ptr_list` module should be made `#[cfg(before_api = "4.3")]`, and the parts that don't have this attribute moved out. After all, the name no longer fits...